code/src/components/NewThought.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
+export const NewThought = ({newThought, setNewThought, handleSubmit, errorMessage}) => {
+
+    const countCharacters = (maxNumber) => {
+        const characters = maxNumber - newThought.length
+        const characterCount = document.getElementById("character-count");
+
+        if (newThought.length > maxNumber) {
+            characterCount.classList.add("warning")
+        }
```

Comment:
I never really understood what the classList thingy is, is it to add a class to the css?

Reply:
Exactly :)